tests/ffi/network/test_largepckts_icmp.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+#Here $timper is allowed percentage delay and $ploss is allowed packet loss
+timper=5
+ploss=0
+echo " "
+echo "Creating Containers*"
```

Comment:
Consider using env vars with defaults
like this 

TIMPER=${TIMPER:-5}
the same for ploss